process/elasticproc/accounts/accountsProcessor_test.go

```diff
@@ -343,7 +344,8 @@ func TestAccountsProcessor_PrepareAccountsMapESDT(t *testing.T) {
 		Properties:      "3032",
 		TokenNonce:      15,
 		Data: &data.TokenMetaData{
-			Creator: "creator",
+			Creator:    "creator",
+			Attributes: []byte{},
```

Comment:
`make([]byte, 0)` instead of literal declaration

Reply:
done